classroom.rb

```diff
@@ -0,0 +1,14 @@
+class Classroom
+  attr_accessor :label
+  attr_reader :students
+
+  def initialize(_lable)
```

Comment:
- [ ] You have a typo here, I think the _lable needs to  be label

Reply:
@GzimAsani,
thanks for your good and valuable feedback, 
I have considered your comment into account.